plugins/mobile.js

```diff
@@ -13,13 +27,28 @@ Plugin to capture navigator.connection.type on browsers that support it
 	if (!connection) {
 		return;
 	}
-	BOOMR.addVar({
-		"mob.ct": connection.type,
-		"mob.bw": connection.bandwidth,
-		"mob.mt": connection.metered
-	});
-
-	if (connection.downlinkMax) {
-		BOOMR.addVar("mob.lm", connection.downlinkMax);
+
+	function setVars() {
+		var k;
+
+		for (k in param_map) {
+			if (k in connection) {
```

Comment:
@bluesmoon I changed this from `connection.navigator.hasOwnProperty(k)`. That _always_ returned `false` in my testing regardless of whether that prop existed or not. I tried in console and got the same result. Do you see any issue with using `in` instead?

Reply:
`in` returns true even for inherited properties.  Also, should this be `connection.navigator`?  If not, then that was the bug.

Reply:
Sorry...that was a typo in the comment only.